e2e/e2e_client_server.py

```diff
@@ -0,0 +1,41 @@
+
+import argparse
+import utils.CallBacks
+from utils.BluetoothHandler import BluetoothHandler
+
+def main():
+    parser = argparse.ArgumentParser()
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument("-i", help="flag indicating if should initiate as a server",
+                    action="store_true")
+    group.add_argument("--serverMac", help='MAC address of the server for bluetooth connection', type=str)
```

Comment:
```suggestion
    group.add_argument("--client", "-c",  type=str, help="Indicates client mode and accepts the MAC address of the server for bluetooth connection as an argument")
```
- Keeping with the double quotes since they seem to be used everywhere else, fine with either though!